backend/__tests__/notifications.js

```diff
@@ -161,4 +161,195 @@ describe('Notifications', () => {
       expect(notifications).toEqual(expectedOutput)
     })
   })
+
+  describe('retrieveAllFollows', () => {
+    it('should return it in the correct format', async () => {
+      // GIVEN two users
+      const password = 'PASSWORD'
+      const randomUsername1 = (Math.random() + 1).toString(36).substring(7)
+      const email1 = `test@${randomUsername1}.com`
+
+      const user1 = await models.users.create({
```

Comment:
It might be better to use `Helper.createUser()` and other functions in the Helper class for these tests to make them more readable